js/actions/windowActions.js

```diff
@@ -664,6 +664,40 @@ const windowActions = {
   },
 
   /**
+   * Dispatches a mute/unmute call to all frames in a provided list (used by TabList).
+   *
+   * @param {Object} framePropsList - List of frame properties to consider
+   * @param {boolean} muted - true if the frames should be muted
+   */
+  muteAllAudio: function (framePropsList, mute) {
+    framePropsList.forEach((frameProps) => {
+      if (mute && frameProps.get('audioPlaybackActive') && !frameProps.get('audioMuted')) {
+        this.setAudioMuted(frameProps, true)
+      } else if (!mute && frameProps.get('audioMuted')) {
+        this.setAudioMuted(frameProps, false)
+      }
+    })
+  },
+
+  /**
+   * Dispatches a mute call to all frames except the one provided.
+   * The provided frame will have its audio unmuted.
+   *
+   * @param {Object} frameToSkip - Properties of the frame to keep audio
+   */
+  muteAllAudioExcept: function (frameToSkip) {
```

Comment:
I'm ok with this but just noting that both blocks are very similar. You could probably treat undefine passed here as no exception and have the other function call this one.